app/dashboard/utils.py

```diff
@@ -72,9 +79,15 @@ def getIPFS():
         raise IPFSCantConnectException("IPFS is not running.  try running it with `ipfs daemon` before this script")
 
 
+@backoff.on_exception(
+    backoff.expo,
+    requests.exceptions.HTTPError,
+    max_tries=5,
+    giveup=bad_request_response)
 def ipfs_cat(key):
     response = ipfs_cat_requests(key)
```

Comment:
i wonder if it would make sense to do a `DBCache` locally of this information.. 

1. for failover reasons... if infura every doesnt properly pin the hashes that gitcoin cares about so the data isnt lost
2. for peformance reasons.. hitting a local DB is much cheaper than a network request